packages/mui-base/src/SelectUnstyled/useSelect.types.ts

```diff
@@ -0,0 +1,112 @@
+import { UseButtonRootSlotProps } from '../ButtonUnstyled';
```

Comment:
Was this file just renamed? It is a bit hard to review. If it is just about renaming, can be do it in a separate PR?